src/extension.ts

```diff
@@ -4,6 +4,7 @@ import * as sander from 'sander';
 // tslint:disable-next-line:no-implicit-dependencies
 import * as vscode from 'vscode';
 import * as wrap from 'wrap-ansi';
+const types = require('conventional-commit-types').types;
```

Comment:
I think its not a good idea to mix and match ESM and commonjs.
I would prefer if you use `import` syntax and provide typescript defintitions for the import if there are none available.